mappings/net/minecraft/client/gui/screen/realms/RealmsSelectFileToUploadScreen.mapping

```diff
@@ -1,21 +1,29 @@
 CLASS net/minecraft/unmapped/C_mlitujqf net/minecraft/client/gui/screen/realms/RealmsSelectFileToUploadScreen
+	FIELD f_gpaeqsoa UPLOAD_WORLD Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_idpqzakq selectedWorld I
 	FIELD f_ihhnexnt uploadButton Lnet/minecraft/unmapped/C_buwziidm;
 	FIELD f_jcmcolvp DATE_FORMAT Ljava/text/DateFormat;
 	FIELD f_liyuktyc HARDCORE Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_lmkqaeqh COMMANDS Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_ltwieqjv serverId J
 	FIELD f_lwngoddz LOGGER Lorg/slf4j/Logger;
 	FIELD f_lxkctjpy parent Lnet/minecraft/unmapped/C_jpqqwpue;
 	FIELD f_tpqpxgar UNABLE_TO_LOAD Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_vynygspi worldSelectionList Lnet/minecraft/unmapped/C_mlitujqf$C_sdcwrgas;
 	FIELD f_wpnzcbpk WORLD_LANG Lnet/minecraft/unmapped/C_rdaqiwdt;
-	FIELD f_xlnecsox levelList Ljava/util/List;
```

Comment:
how did I let this pass :sob:

Reply:
They are from yarn probably, a lot of the files had mappings back from when we changed to hashed

Reply:
mmm I don't think so I purged level personally a while ago

Reply:
![image](https://github.com/QuiltMC/quilt-mappings/assets/47987888/16d9c3d6-b8af-4491-8f27-91b36d813b51)
even with an original yarn commit just below

> mmm I don't think so I purged level personally a while ago

you missed one :p

Reply:
sobbing....